public/locales/en/zod.json

```diff
@@ -0,0 +1,22 @@
+{
+    "errors": {
+        "default": "This field is invalid",
+        "required": "This field is required",
+        "string": {
+            "startsWith": "This field must start with {{startsWith}}",
+            "endsWith": "This field must end with {{endsWith}}",
+            "includes": "This field must include {{includes}}"
+        },
+        "too_small": {
+            "string": "This field must be at least {{minimum}} characters long",
+            "number": "This field must be greater than or equal to {{minimum}}"
+        },
+        "too_big": {
+            "string": "This field must be at most {{maximum}} characters long",
+            "number": "This field must be less than or equal to {{maximum}}"
```

Comment:
Please use `camelCase` and not snake_case.. You're using camelCase bellow for custom.passwordMatch